pkg/services/provisioning/dashboards/types.go

```diff
@@ -91,13 +97,19 @@ func mapV0ToDashboardAsConfig(v0 []*DashboardsAsConfigV0) []*DashboardsAsConfig
 		})
 	}
 
-	return r
+	return r, nil
 }
 
-func (dc *DashboardAsConfigV1) mapToDashboardAsConfig() []*DashboardsAsConfig {
+func (dc *DashboardAsConfigV1) mapToDashboardAsConfig() ([]*DashboardsAsConfig, error) {
```

Comment:
As mentioned before, I think it's better to be consistent and mirror the plural form of DashboardsAsConfig:
```suggestion
func (dc *DashboardAsConfigV1) mapToDashboardsAsConfig() ([]*DashboardsAsConfig, error) {
```